OneCommunityGlobal/HighestGoodNetworkApp
# Kaikanes console warning bugfix

# Description
I notice this bug seems to be unique to my profile.
I've attempted to re-create the bug by making a new volunteer user or Admin user with now luck
It appears the bug has something to do with the projects and tasks array having the same key value
Not sure why its only happening on this profile but these changes are minimal.
## Related PRS (if any):
This frontend PR only 
…

## Main changes explained:
- Added unique string before the key
- Added comments where I changed lines
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as kaikane3457@hotmail.com
4. use my password:  A5-9#<B"PX=WkK$8
4. go to dashboard -> open console
5. should see 
![Weird bug](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/1bf48740-572f-4001-b116-ce4f6ece20ad)

## Screenshots or videos of changes:
With these changes the warning went away.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/9c2a4ed1-622b-4088-af35-0719eac2df61)

## Note:
Include the information the reviewers need to know.
